index.src.html

```diff
@@ -290,6 +290,16 @@
     </ul>
   </section>
 
+  <section algorithm="'create a credential' internal method">
+    <dfn for="Credential" method export>\[[Create]](data)</dfn> is called with a
+    {{CredentialCreationData}}, and returns either a {{Credential}}, if one can be created, or
```

Comment:
`[[Create]]` is synchronous, and `create()`, which wraps `[[Create]]` in a developer exposed way, is not. That seemed to make things simpler to write. *shrug* If you disagree, then we'd need to change the other internal methods as well.